### PR TITLE
Improve addons names and descriptions

### PR DESCRIPTION
**Pull request by nbessi**

### CODE REVIEW

File: l10n_ch_bank/__openerp__.py [hunk 1]
```diff
@@ -27,9 +27,16 @@
...
-    Swiss banks list
+{"name": "Switzerland - Bank list",
+ "summary": "Banks names, addresses and BIC codes",
+ "description": """
+Swiss bank list
+===============
+This module will load all Swiss banks in OpenERP with their name, address and BIC code to
+ease the input of bank account.
+
+It is not mandatory to use OpenERP in Switzerland, but can improve the user experience.
```
Comment (pedrobaeza) [inline l10n_ch_bank/__openerp__.py]: Instead `to use` must be `for using`

File: l10n_ch_base_bank/__openerp__.py [hunk 2]
```diff
@@ -19,11 +19,18 @@
...
-================================================
-This addons will add different bank type required by
-DTA, and ESR system in order to manage duality of Post and bank systems
+Swiss bank type and fields
+==========================
+
+This addons will add different bank types required by specific swiss electronic
+payment like DTA and ESR. It allows to manage both Post and Bank systems.
+
+It'll perform some validation when entring bank account number or ESR number in invoice
```
Comment (pedrobaeza) [inline l10n_ch_base_bank/__openerp__.py]: Respect PEP8 80 cols max length

File: l10n_ch_scan_bvr/__openerp__.py [hunk 3]
```diff
@@ -19,15 +19,16 @@
...
+{"name": "Switzerland - Scan ESR/BVR to create invoices",
  "description": """
-create invoices from BVR code
+Scan ESR/BVR to create supplier invoices
+========================================
 
 This module works with C-channel or other OCR scanner.
 
-It helps you to create an invoice directly from the BVR Code. Find the menu entry
+It helps you to create an invoice directly from the ESR/BVR Code. Find the menu entry
```
Comment (pedrobaeza) [inline l10n_ch_scan_bvr/__openerp__.py]: PEP8 80 cols limit

File: l10n_ch_sepa/__openerp__.py [hunk 4]
```diff
@@ -20,21 +20,23 @@
...
     "version": "1.0",
     "category": "Finance",
     "description": """
-SEPA payments
-=============
+Swiss electronic payment (SEPA)
+===============================
 
-pain.001 for Credit Transfert Initiation
+This addons allows you to generate SEPA electronic payment file for Switzerland.
```
Comment (pedrobaeza) [inline l10n_ch_sepa/__openerp__.py]: PEP8 80 cols limit

File: l10n_ch_zip/__openerp__.py [hunk 5]
```diff
@@ -27,13 +27,21 @@
...
  'description': """
-better zip data for Switzerland
-###############################""",
+Swiss postal code (ZIP) list
+============================
+
+This module will load all Swiss postal codes (ZIP) in Odoo to
+ease the input of partners.
+
+It is not mandatory to use Odoo in Switzerland, but can improve the user experience.
```
Comment (pedrobaeza) [inline l10n_ch_zip/__openerp__.py]: `for using` and PEP8 limit

File: l10n_ch_sepa/__openerp__.py [hunk 6]
```diff
@@ -20,23 +20,26 @@
...
 
-This module designed for Swiss payment is based on the norm for euro payments
-but implements the special Swiss extensions.
+This module designed for Swiss payment is also generic for euro payments
+as the Swiss standards are slightly different.
 It intends to be reusable to accept new definition of specific standards
 by country.
 
-TODO: Needs to be fully tested with a FI
+It currently supports the "pain.001$2 norm for Credit Transfert Initiation.
```
Comment (yvaucher) [inline l10n_ch_sepa/__openerp__.py]: "pain.001" instead of "pain.001$2

**Comment by yvaucher** [comment]
:+1:

**Comment by guewen** [comment]
Squashing commits could be nice, but :+1:

**Pull request merged**